Commands.adoc

```diff
@@ -0,0 +1,226 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you do `vagrant up` and run an ADB box, as a developer you may want to
+customize or set up your development environment to optimally utilize the ADB as
+ per your requirements.
+
+The vagrant-service-manager plugin enables you to check and tweak your
+container development environment.
+
+'''
+toc::[]
+'''
+
+You can use the vagrant service manager to set up your environment variables and
```

Comment:
"you can use vagrant-service-manager". Let's stick to one way of writing this. So far we use dashes.